country-a-service/cda-generator/src/main/java/dk/sundhedsdatastyrelsen/ncpeh/cda/Utils.java

```diff
@@ -18,4 +20,19 @@ public static String cdaDateTime(TemporalAccessor time) {
     public static String cdaDate(TemporalAccessor time) {
         return DateTimeFormatter.ofPattern("yyyyMMdd", Locale.ROOT).format(time);
     }
+
+    /**
+     * Convert XMLGregorianCalendar to OffsetDateTime
+     */
+    public static OffsetDateTime convertToOffsetDateTime(XMLGregorianCalendar xmlGregorianCalendar) {
```

Comment:
They do mostly the same, just using small changes in the calls. I have merged them, and left the method in the Utils class (in CDA, we also have one in service, but the CDA generation cannot see that since that would cause circular references - we might want to consider merging the Utils in one place)